### PR TITLE
Manual on how to get started with contributing to the docs

### PR DESCRIPTION
**Pull request by groggled**
I added a simple manual on how to contribute to the docs mainly covering git.

### CODE REVIEW

File: content/en/docs/contributing/documentation.md [hunk 1]
```diff
@@ -13,4 +13,29 @@ weight: 3300
...
+# Getting started
+
+For contributing you will need the following:
+- git installed, this will most likely already be the case
+- a GitHub account
+- a text or markdown editor like VSCode
+- some knowledge of markdown ([guide to markdown](https://www.markdownguide.org/basic-syntax))
+- nodejs (optional) for previewing your changes on a local website
+
+To set up your editing environment you will need to fork and clone the [repository](https://github.com/an-anime-team/docs). The follwing will show you how to create your own copy (fork) and download (clone) it.
```
Comment (Mar0xy) [inline content/en/docs/contributing/documentation.md]: You seem to have made a typo in the word "following"

File: content/en/docs/contributing/documentation.md [hunk 2]
```diff
@@ -13,4 +13,29 @@ weight: 3300
...
 
-- Date format: [ISO](https://greenwichmeantime.com/articles/clocks/iso)
+# Getting started
+
+For contributing you will need the following:
+- git installed, this will most likely already be the case
+- a GitHub account
+- a text or markdown editor like VSCode
+- some knowledge of markdown ([guide to markdown](https://www.markdownguide.org/basic-syntax))
+- nodejs (optional) for previewing your changes on a local website
```
Comment (Mar0xy) [inline content/en/docs/contributing/documentation.md]: We recommend using nodejs when adding new pages and I think this should be communicated

**Review by Mar0xy — CHANGES_REQUESTED** [review]
I reviewed your edit and so far it is great besides 2 small issues.

File: content/en/docs/contributing/documentation.md (hunk 2 above)
Comment (groggled) [reply content/en/docs/contributing/documentation.md]: I don't know how to embed the changes, but I changed it to:
> - nodejs is also strongly recommended, especially when creating new pages to see how changes appear on the wiki

I hope that resolves it

File: content/en/docs/contributing/documentation.md [hunk 3]
```diff
@@ -13,4 +13,29 @@ weight: 3300
...
+- some knowledge of markdown ([guide to markdown](https://www.markdownguide.org/basic-syntax))
+- nodejs is also strongly recommended, especially when creating new pages to see how changes appear on the wiki
+
+To set up your editing environment you will need to fork and clone the [repository](https://github.com/an-anime-team/docs). The following will show you how to create your own copy (fork) and download (clone) it.
+
+Open the repository in a browser, log in to GitHub and click on "Fork" in the top right. Just click on "Create fork" in the next step and you will have forked the repository!
+
+To clone your fork type `git clone https://github.com/[your GitHub username]/docs.git` into a terminal and git will create a directory named docs and download the repository there.
+
+Now you just need to edit something!
```
Comment (Mar0xy) [inline content/en/docs/contributing/documentation.md:23]: Put the information that you can run the wiki locally by doing `npm run start` for seeing the changes live somewhere

Comment (groggled) [reply content/en/docs/contributing/documentation.md:23]: Sure, just added it

**Review by Mar0xy — APPROVED** [review]

**Pull request merged**